src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -8,12 +8,149 @@
 import orchestrator
 from .fs_util import create_pool
 
+from typing import List
+
 log = logging.getLogger(__name__)
 
+def available_clusters(mgr):
+    completion = mgr.describe_service(service_type='nfs')
+    mgr._orchestrator_wait([completion])
+    orchestrator.raise_if_exception(completion)
+    # Remove 'ganesha-' prefixes from cluster ids.
```

Comment:
It'd be helpful to give an example in a comment of what `completion.result` looks like at this point.

Reply:
Done